app/controllers/api/v2/registration_commands_controller.rb

```diff
@@ -21,6 +21,7 @@ class RegistrationCommandsController < V2::BaseController
         param :update_packages, :bool, desc: N_("Update all packages on the host")
         param :repo, String, desc: N_("Repository URL / details, for example for Debian OS family: 'deb http://deb.example.com/ buster 1.0', for Red Hat and SUSE OS family: 'http://yum.theforeman.org/client/latest/el8/x86_64/'")
         param :repo_gpg_key_url, String, desc: N_("URL of the GPG key for the repository")
+        param :download_utility, String, desc: N_("The download utility to use for host registration")
```

Comment:
I think you can limit it with a validator. See https://github.com/Apipie/apipie-rails#parameter-description as well.